src/core/annotation.js

```diff
@@ -1741,11 +1741,13 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
     // The parent field's `V` entry holds a `Name` object with the appearance
     // state of whichever child field is currently in the "on" state.
     const fieldParent = params.dict.get("Parent");
-    if (isDict(fieldParent) && fieldParent.has("V")) {
-      const fieldParentValue = fieldParent.get("V");
-      if (isName(fieldParentValue)) {
-        this.parent = params.dict.getRaw("Parent");
-        this.data.fieldValue = this._decodeFormValue(fieldParentValue);
+    if (isDict(fieldParent)) {
+      this.parent = params.dict.getRaw("Parent");
+      if (fieldParent.has("V")) {
+        const fieldParentValue = fieldParent.get("V");
+        if (isName(fieldParentValue)) {
+          this.data.fieldValue = this._decodeFormValue(fieldParentValue);
+        }
       }
```

Comment:
Can't this be simplified a little bit, since having a separate `fieldParent.has("V")` check seems redundant now?
```suggestion
      const fieldParentValue = fieldParent.get("V");
      if (isName(fieldParentValue)) {
        this.data.fieldValue = this._decodeFormValue(fieldParentValue);
      }
```